lib/Parser/Parser.cpp

```diff
@@ -712,6 +712,54 @@ bool Parser::parseLine(std::string &ErrStr) {
         if (!LHS)
           return false;
 
+        llvm::APInt DemandedBitsVal = APInt(LHS->Width, 0, false);
+        llvm::APInt ConstOne(LHS->Width, 1, false);
+        if (CurTok.K == Token::OpenParen) {
+          if (!consumeToken(ErrStr))
+            return false;
+          if (CurTok.K != Token::Ident) {
+            ErrStr = makeErrStr("invalid demandedBits token type");
```

Comment:
don't tell the user about token type, they don't care about tokens, say what was expected and what was seen